### PR TITLE
Add/Change Javascript license notice

### PR DESCRIPTION
**Pull request by CalculationPaper**
Add missing js file and clarify the MIT license variant

### CODE REVIEW

File: src/invidious/views/licenses.ecr [hunk 1]
```diff
@@ -23,6 +23,20 @@
...
+
+            <td>
+                <a href="https://www.gnu.org/licenses/agpl-3.0.html">AGPL-3.0</a>
+            </td>
+
+            <td>
+                <a href="/js/helpers.js?v=<%= ASSET_COMMIT %>"><%= translate(locale, "source") %></a>
+            </td>
+        </tr>
+
```
Comment (SamantazFox) [inline src/invidious/views/licenses.ecr:39]: There is no such file as `helpers.js` ^^
The license for [_helpers.js](https://github.com/iv-org/invidious/blob/master/assets/js/_helpers.js) is already in right above :)

File: src/invidious/views/licenses.ecr [hunk 2]
```diff
@@ -169,7 +183,7 @@
             </td>
 
             <td>
-                <a href="https://choosealicense.com/licenses/mit/">MIT</a>
+                <a href="https://choosealicense.com/licenses/mit/">Expat</a>
```
Comment (SamantazFox) [inline src/invidious/views/licenses.ecr:186]: I'm not sure why you'd put `Expat` in here? What does that mean?

File: src/invidious/views/licenses.ecr [hunk 3]
```diff
@@ -253,7 +267,7 @@
             </td>
 
             <td>
-                <a href="https://choosealicense.com/licenses/mit">MIT</a>
+                <a href="https://choosealicense.com/licenses/mit">Expat</a>
```
Comment (SamantazFox) [inline src/invidious/views/licenses.ecr:270]: https://github.com/videojs/videojs-vr/issues/264

File: src/invidious/views/licenses.ecr (hunk 1 above)
Comment (CalculationPaper) [reply src/invidious/views/licenses.ecr]: Oh, my fault, sorry. I actually meant handlers.js I guess. Looks nearly the same and doesn't sound too different. Can use it for rhymes. :D

**Comment by CalculationPaper** [comment]
I forgot too mention that changing "MIT" to "Expat" or "X11" makes it complies to the LibreJS browser-extension. According to the FSF there are multiple licenses the Massachusetts Institute of Technology (MIT) used/uses, so calling it "MIT license" could be misleading. A prominent example for "MIT licenses" are the "Expat" and "X11" variants and there is a notable difference between them. The "X11 License" have an advertising clause.

**Comment by SamantazFox** [comment]
I feel like it's some "flame war" thing. I've always seen the name "MIT license" being used.

But in all cases, we should use what mentioned projects use. For instance, it should be "MIT" for `videojs-vtt-thumbnails`, `videojs-markers` and `videojs-http-source-selector`.

**Comment by CalculationPaper** [comment]
You are free to call it how you want. However, LibreJS's machine readable format seems to require the distinction between those two different MIT Licenses.

**Review by TheFrenchGhosty — APPROVED** [review]
I don't really know the details, but if it make Invidious compatible/compliant with LibreJS, I'm for it.

**Pull request merged**